code/modules/clothing/masks/gasmask.dm

```diff
@@ -162,11 +162,6 @@
 	item_state = "sexymime"
 	burn_state = FLAMMABLE
 
-/obj/item/clothing/mask/gas/death_commando

```

Comment:
This was in the code, but isn't used and doesn't even have a working icon.

Reply:
Uses the same icon as the cyborg visor mask thing, never seen this object used though.